Include/cpython/unicodeobject.h

```diff
@@ -445,6 +437,16 @@ enum PyUnicode_Kind {
         (0xffffU) :                                                     \
         (0x10ffffU)))))
 
+/* Py_DEPRECATED(3.3) */
+#define PyUnicode_WSTR_LENGTH(op) _PyUnicode_WSTR_LENGTH_impl((PyObject*)op)
```

Comment:
nitpick: please move it after _PyUnicode_WSTR_LENGTH_impl(), since the macro calls _PyUnicode_WSTR_LENGTH_impl().